site-src/guides/epp-configuration/prefix-aware.md

```diff
@@ -0,0 +1,89 @@
+# Prefix Cache Aware Plugin Configuration
+
```

Comment:
is the link not pointing to main intentionally?

Reply:
this is a permanent link. Main can change.